src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -122,34 +111,38 @@ public void assertContainsAtLeast(@NotNull ItemStack item, int amount)
 	 * @param item The item to check for.
 	 * @return The number of times the item is present in this inventory.
 	 */
-	public int getNumberOfItems(@NotNull ItemStack item)
-	{
+	public int getNumberOfItems(@NotNull ItemStack item) {
```

Comment:
You broke the formatting here.
```suggestion
	public int getNumberOfItems(@NotNull ItemStack item)
	{
```